pkg/scheduler/api/cluster_info.go

```diff
@@ -20,9 +20,10 @@ import "fmt"
 
 // ClusterInfo is a snapshot of cluster by cache.
 type ClusterInfo struct {
-	Jobs   map[JobID]*JobInfo
-	Nodes  map[string]*NodeInfo
-	Queues map[QueueID]*QueueInfo
+	Jobs          map[JobID]*JobInfo
+	Nodes         map[string]*NodeInfo
+	Queues        map[QueueID]*QueueInfo
+	NamespaceInfo map[string]*NamespaceInfo
```

Comment:
xref: https://github.com/volcano-sh/volcano/pull/193#discussion_r285676939

move All NamespaceInfo to UserInfo